cmake/protobuf.cmake

```diff
@@ -8,11 +8,13 @@ unset(_absl_provider)
 unset(_build_shared_libs)
 unset(_source_subdir)
 
+# -- Following is not needed after move to protobuf v25.0
```

Comment:
We can dispense with this section altogether, now that you've update the default version of Protobuf. It was only necessary because the default version antedated the move from `cmake` to the main directory.

Reply:
Ack. Removed it.